src/services/AuthService.tsx

```diff
@@ -121,35 +153,24 @@ const Auth = (props: AuthProps) => {
 			.then((response: unknown) => {
 				setUser({ name: username });
 				const {
-					message: { access_exp: accessExp }
-				} = response as { message: { access_exp: number } };
+					content: { access_exp }
+				} = response as ResAuthLogin;
 
-				const refreshDelay = Math.max((accessExp - new Date().getTime()) / 2, 0);
+				const refreshDelay = Math.max((access_exp - new Date().getTime()) / 2, 1000);
 
 				setRefreshInterval(refreshDelay);
 			})
-			.catch((error: HTTPError) => {
-				console.error(error);
-				console.log(error?.response);
-				switch (error?.response?.status) {
-					case 401:
-						alert('Username or password is wrong');
-						break;
-				}
-			});
+			.catch((_: HTTPError) => {});
 	};
 
 	const logout = () => {
 		kyRef.current
 			.delete(`${BACKEND_URL}/auth/logout`)
 			.json()
-			.then((response: unknown) => {
+			.then(() => {
 				setUser(null);
 			})
-			.catch((error: HTTPError) => {
-				console.error(error);
-				console.log(error?.response);
-			});
+			.catch((_: HTTPError) => {});
```

Comment:
why did you remove error logging?

Reply:
Error logging is handled in the callback after every response 
src\services\AuthService.tsx
![obraz](https://user-images.githubusercontent.com/26521377/145875551-70d458cc-185e-4c9b-9458-6c0b33a46999.png)